metrics-operator/go.mod

```diff
@@ -122,4 +121,9 @@ require (
 
 )
 
-replace golang.org/x/net => golang.org/x/net v0.17.0
+replace (
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.45.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.45.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0
+	golang.org/x/net => golang.org/x/net v0.17.0
```

Comment:
do we still need this replacement for x/net? 🤔

Reply:
It can be removed https://github.com/keptn/lifecycle-toolkit/pull/2312/files#diff-11599b6521406cb805ac12aedd505ef87d49c9083c480b7eed73afa728d6a05cR19

Reply:
thanks, good catch - i have removed the replace fr x/net